db/migrate/20170227155843_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.belongs_to :user
+      t.belongs_to :article
```

Comment:
You can add option `foreign_key: true` because the comment cannot exist without an article, also you need set `index` option to this column because it is the most popular field for selection.